README.md

```diff
@@ -172,6 +181,9 @@ Installation Instructions
 
      - `scale_cluster_gui`: User defined node designation for Spectrum Scale GUI. It
        can be either true or false.
+       
+     - `is_protocol_node`: User defined node designation for Spectrum Scale Protocol. It
```

Comment:
This variable doesn't follow the naming of the others... I'd suggest to use a consistent naming scheme, such as: `scale_cluster_ces` or `scale_role_ces` or something like this...

Also - I'd suggest to either use the term **CES** or use the term **protocol** and then use it consistently...